packages/help-center/src/components/help-center-chat-history.tsx

```diff
@@ -67,6 +67,7 @@ const Conversations = ( {
 						key={ conversation.id }
 						message={ lastMessage }
 						conversation={ conversation }
+						unreadMessages={ unreadMessages }
```

Comment:
You can leave it that way if you prefer, but since we already pass the whole `conversation` object, it would be really easy to retrieve the number of unread messages from inside `HelpCenterSupportChatMessage` components - which would simplify any calling code.

Reply:
From all the suggestions I'm going to leave this one, somehow I like it more modular too